import sys
import traceback
def deep_error():
#    try:
    print(1/0)
#    except:
#        print(sys.exc_info())
#        print(sys.exc_info()[2].tb_frame.f_code.co_filename)
#        print(sys.exc_info()[2].tb_lineno)
#        print(traceback.extract_stack())
#        print(traceback.format_tb(exc_traceback))
